src/routes/api.php

```diff
@@ -46,6 +52,13 @@
     Route::get('/units', [DepartmentUnitController::class, '__invoke']);
     Route::get('/document-types', [DocumentTypeController::class, '__invoke']);
     Route::get('/document-classified', [DocumentClassifiedController::class, '__invoke']);
+    Route::get('/klasifikasi-arsip', [EcoKlasifikasiArsipController::class, '__invoke']);
+    Route::get('/tujuan', [EcoTujuanController::class, '__invoke']);
+    Route::get('/jenis-naskah', [EcoJenisNaskahController::class, '__invoke']);
+    Route::get('/sifat-naskah', [EcoSifatNaskahController::class, '__invoke']);
+    Route::get('/jenis-lampiran', [EcoJenisLampiranController::class, '__invoke']);
+    Route::post('/kirim-naskah', [EcoKirimNaskahController::class, '__invoke']);
```

Comment:
kang @syaifulmuflih95 kang @bioe56 punten, berhubung Mobile API ini juga digunakan untuk beberapa API lain termasuk API e-logistik, agar lebih clear bagaimana kalau kita tambahan prefix ke route2 ini agar terpisah dari endpoint-endpoint lain? usulan saya:

```suggestion
   Route::prefix('ekosistem_eoffice')->group(function () {
    Route::get('/klasifikasi-naskah', [EcoKlasifikasiArsipController::class, '__invoke']);
    Route::get('/tujuan', [EcoTujuanController::class, '__invoke']);
    Route::get('/jenis-naskah', [EcoJenisNaskahController::class, '__invoke']);
    Route::get('/sifat-naskah', [EcoSifatNaskahController::class, '__invoke']);
    Route::get('/jenis-lampiran', [EcoJenisLampiranController::class, '__invoke']);
    Route::post('/kirim-naskah', [EcoKirimNaskahController::class, '__invoke']);
    });
```

👆 ini saya juga mengusulkan agar nama route `/klasifikasi-arsip` diubah namanya jadi `/klasifikasi-naskah` agar seragam dengan penamaan route yang lain. Untuk nama controller nya sepertinya tidak apa2 tidak perlu dirubah. 

Mangga akang2 kalau ada response 🙏